extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -242,18 +247,14 @@ export abstract class AzureAuth implements vscode.Disposable {
 			Logger.verbose(`Fetching tenants with uri: ${tenantUri}`);
 			let tenantList: string[] = [];
 
-			const tenantResponse = await this.httpClient.sendGetRequestAsync<AzureListOperationResponse<TenantResponse[]> | ErrorResponseBodyWithError>(tenantUri, {
-				headers: {
-					'Content-Type': 'application/json',
-					'Authorization': `Bearer ${token}`
-				}
-			});
+			const tenantResponse = await this.makeGetRequest(tenantUri, token);
```

Comment:
This should be using `makeGetRequest<ResponseData>`, right? Then you shouldn't need the cast below.